server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -1034,6 +1048,82 @@ func LookupInvocation(env environment.Env, ctx context.Context, iid string) (*in
 	return invocation, nil
 }
 
+func GetCacheScoreCard(ctx context.Context, env environment.Env, req *capb.GetCacheScoreCardRequest) (*capb.GetCacheScoreCardResponse, error) {
```

Comment:
can we move this scorecard searching / sorting logic to its own file (called scorecard.go?)

Reply:
Done